RepositoryCompilerTests/Unit/MetricCalculationTests.cs

```diff
@@ -163,6 +163,22 @@ public void Calculates_max_nested_blocks_in_class()
             firstClass.Metrics[CaDETMetric.CMNB].ShouldBe(4);
         }
 
+        [Fact]
+        public void Calculates_number_of_dependencies_in_class()
```

Comment:
Since you added fields, variables, and return types, you should expand the other TestSuite in this package to check if your types are correctly determined. Make sure your tests include classes with the same name.

At one point we will need to refactor RepoCompiler tests to have better test data (the factory has gotten out of hand) - e.g., a complete project that we can thoroughly test with all edge cases.